src/Behat/Config/Config.php

```diff
@@ -5,14 +5,27 @@
 namespace Behat\Config;
 
 use Behat\Testwork\ServiceContainer\Exception\ConfigurationLoadingException;
+use PhpParser\BuilderFactory;
+use PhpParser\Node;
+use PhpParser\Node\Name\FullyQualified;
 
 use function is_string;
 
-final class Config implements ConfigInterface
+final class Config implements ConfigInterface, ConfigConverterInterface
```

Comment:
Each Config object that knows how to convert itself to a php expression implements this new interface